openproblems/tasks/dimensional_reduction/metrics/root_mean_square_error.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import scipy as sp
+from scipy import spatial
+import pandas as pd
+from ....tools.decorators import metric
+
+
+@metric(metric_name="compare_rmse", maximize=True)
```

Comment:
Method name should be human readable, e.g. Root Mean Squared Error

Reply:
fixed thanks for the suggestion